notebook/tests/selenium/test_empty_arrows_keys.py

```diff
@@ -0,0 +1,13 @@
+# selenium test version for 'empty_arrow_keys.js'
+
+
+def remove_cells(notebook):
+    for i in notebook.cells:
+        notebook.delete_cell(notebook.index(i))
+    return True
+
+def test_empty_arrows_keys(notebook):
+    # delete all the cells
+    notebook.trigger_keydown('up')
+    notebook.trigger_keydown('down')
+    assert remove_cells(notebook);
```

Comment:
The original JS version of the test removes the cells before doing the up/down keys. So it's testing up/down inside an empty notebook. But to be honest, I have no idea what that test is meant to achieve. You can't even really have an empty notebook - if you delete the last cell, it creates a new one for you. Like Nature, Jupyter abhors a vacuum.

Maybe it would be more use to test that behaviour instead - delete all cells, check there's a new one there.

Reply:
Hmm, I agree that test was testing minimal behavior. Yes, Sure, I can add do this check.